docs/connector-development/config-based/connector-builder-compatibility.md

```diff
@@ -0,0 +1,210 @@
+
+## Is the integration a REST API returning a collection of records synchronously?
+
+Is the integration a REST API? In doubt, look for the word “REST” in the documentation.
+
+Rook at the shape of the resource path.
+
+Taking the [Congress API](https://api.congress.gov/#/bill) as an example,
+
+`GET /bill`
+
+Indicates the records can be retrieved by submitting a GET request to the /bill path.The sample response shows that the response returns a collection of records, so the Congress API is REST API returning a collection of records.
+
+Sample response:
+```
+{
+  "bills":[
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Became Public Law No: 117-108."
+      },
+      "number":"3076",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Postal Service Reform Act of 2022",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:27:05Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3076?format=json"
+    },
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Read twice. Placed on Senate Legislative Calendar under General Orders. Calendar No. 343."
+      },
+      "number":"3599",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Federal Rotational Cyber Workforce Program Act of 2021",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:41:50Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3599?format=json"
+    }
+  ]
+}
+```
+
+Some endpoints are parameterized. An example of such endpoint is the [Congress API’s bill by congress endpoint](https://api.congress.gov/#/bill/bill_list_by_congress).
+`GET /bill/:congress`
+
+These endpoints are also valid synchronous REST endpoints.
+
+This differs from the [Amazon Ads reports endpoint](https://advertising.amazon.com/API/docs/en-us/info/api-overview)), which returns a report ID, which will be generated asynchronously by the source. Because the reports need to be downloaded separately, this is not a synchronous REST API.
+
+Examples:
+- Yes: [Congress API](https://api.congress.gov/#/)
+- No: [Amazon Ads](https://advertising.amazon.com/API/docs/en-us/info/api-overview)
+
+If the integration is not a synchronous REST API, use the Python CDK.
+
+## What type of authentication is required?
+Look up the authentication mechanism in the API documentation, and identify which type it is.
+
+### Basic HTTP
+Are requests authenticated using the Basic HTTP authentication method? You can search the documentation page for one of the following keywords
+- "Basic Auth"
+- "Basic HTTP"
+- "Authorization: Basic"
+- "Base64"
+
+Example: [Greenhouse](https://developers.greenhouse.io/harvest.html#introduction)
+
+### API Key
+Are requests authenticated using an API key injected either as a query parameter or as a request header?
```

Comment:
> injected either as a query parameter or as a request header

The current API Key auth method in the connector builder/low-code CDK only supports injecting the API key into the request header at the moment. 

Based on what you wrote here, it sounds like we may want to allow users to inject this into a query parameter instead? They can technically do this by adding a Request Parameter field to a stream, but it feels like we should make this a first-class option in the Auth component if it is common

Reply:
I think the description is valid because we do allow for setting the API key as a query param with the workaround, but I agree with fixing the issue instead of documenting the workaround